README.md

```diff
@@ -16,15 +16,35 @@ Want to get paid for your contributions to `eslint-config-amex`?
 
 * [Usage](#-usage)
 * [Contributing](#-contributing)
-* [License](#-license)
-* [Code of Conduct](#-code-of-conduct)
+* [License](#️-license)
+* [Code of Conduct](#️-code-of-conduct)
 
 ## 🤹‍ Usage
 
 ### Installation
 
+
+```bash
+npm install --save-dev eslint eslint-config-amex
+```
+
+The default config provides support for React applications. ESLint configs are also provided for:
+
+* Jest
+* Prettier
+
+Install required peer dependencies before [extending your ESLint config](#extend-your-eslintrc):
+
+Jest:
+
+```bash
+npm install --save-dev typescript eslint-plugin-jest eslint-plugin-jest-dom
```

Comment:
I might be out of the loop here, but why is `typescript` needed for `jest` support?

Reply:
`eslint-plugin-jest` has a direct dependency on a package called `@typescript-eslint/experimental-utils` which eventually pulls in a package that depends on `tsutils`, which requires `typescript`:

https://github.com/jest-community/eslint-plugin-jest/issues/339

I think that their removal of `typescript` from `dependencies` of that package was a mistake, as the plugin cannot function without it and the warning is buried under a transient dependency. I'll probably raise a PR over there but I'd rather `eslint-config-amex` not bring TypeScript along with it in the default configurations.